### PR TITLE
Change: From default dockerhub to community registry

### PR DESCRIPTION
**Pull request by gabrielschowe**
## What
Change: Added registries.

## Why
The old registries will soon be outdated, so it will be changed to the new one.
Not all are currently uploaded.

## References
[DEVOPS-1130](https://jira.greenbone.net/browse/DEVOPS-1130)

## Checklist

- [ ] [Changelog](src/changelog.md) entry

### CODE REVIEW

File: src/_static/docker-compose-22.4.yml [hunk 1]
```diff
@@ -115,7 +115,7 @@ services:
 
   # shows logs of openvas
   openvas:
-    image: greenbone/openvas-scanner:stable
+    image: greenbone/openvas-scanner
```
Comment (y0urself) [inline src/_static/docker-compose-22.4.yml:118]:
```suggestion
    image: registry.community.greenbone.net/community/openvas-scanner
```

File: src/_static/docker-compose-22.4.yml [hunk 2]
```diff
@@ -160,7 +160,7 @@ services:
           - openvasd
 
   ospd-openvas:
-    image: greenbone/ospd-openvas:stable
+    image: registry.community.greenbone.net/community/ospd-openvas:edge
```
Comment (y0urself) [inline src/_static/docker-compose-22.4.yml:163]:
```suggestion
    image: registry.community.greenbone.net/community/ospd-openvas
```

Even if `edge` works, we do not want to use it in this compose file ... it should just be latest or stable I guess.

File: src/_static/docker-compose-22.4.yml [hunk 3]
```diff
@@ -86,7 +86,7 @@ services:
         condition: service_completed_successfully
 
   gsa:
-    image: greenbone/gsa:stable
+    image: registry.community.greenbone.net/community/gsa:edge
```
Comment (y0urself) [inline src/_static/docker-compose-22.4.yml:89]:
```suggestion
    image: registry.community.greenbone.net/community/gsa:stable
```

File: src/_static/docker-compose-22.4.yml [hunk 4]
```diff
@@ -1,65 +1,65 @@
 services:
   vulnerability-tests:
-    image: greenbone/vulnerability-tests
+    image: registry.community.greenbone.net/community/vulnerability-tests:stable
```
Comment (y0urself) [inline src/_static/docker-compose-22.4.yml:3]: Please just leave the tags as they where ... this means in this case: No tag.

```suggestion
    image: registry.community.greenbone.net/community/vulnerability-tests
```

File: src/_static/docker-compose-22.4.yml [hunk 5]
```diff
@@ -199,7 +199,7 @@ services:
         condition: service_completed_successfully
 
   gvm-tools:
-    image: greenbone/gvm-tools
+    image: registry.community.greenbone.net/community/gvm-tools:stable
```
Comment (y0urself) [inline src/_static/docker-compose-22.4.yml:202]: no tag

**Review by y0urself — APPROVED** [review]

**Review by k-schlosser — APPROVED** [review]

**Pull request merged**